ldap_auth_provider.py

```diff
@@ -13,38 +13,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import logging
+import ssl
 from typing import Optional
 
+from pkg_resources import parse_version
 from twisted.internet import threads
 
-
 import ldap3
 import ldap3.core.exceptions
-
-import ssl
-
-import logging
 import synapse
 
-from pkg_resources import parse_version
-
-
 __version__ = "0.1.4"
 
 
+# ldap3 v2 changed ldap3.AUTH_SIMPLE -> ldap3.SIMPLE
 try:
-    import ldap3
-    import ldap3.core.exceptions
```

Comment:
given we import `ldap3` and `ldap3.core.exceptions` above, this and the `if not ldap3` below seem redundant.